Cartfile

```diff
@@ -1 +1 @@
-github "CanalTP/NavitiaSDK_ios" == 1.3.1
+github "CanalTP/NavitiaSDK_ios" == 1.3.2
```

Comment:
There is no version `1.3.2` for `NavitiaSDK_ios`. Can you please put it back to version `1.3.1`?

Reply:
oups...